debezium-connector-mysql/pom.xml

```diff
@@ -57,6 +57,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.mariadb.jdbc</groupId>
+            <artifactId>mariadb-java-client</artifactId>
```

Comment:
I am not sure about this but maybe should drivers be `provided` scope?!

Reply:
We've always shipped all open-source drivers with the connectors to simplify the installation for users.  If a user needs or requires a specific driver, they're free to replace the driver as needed.

Reply:
Yes, both drivers hould be shipped, esp. if they can co-exist side-by-side.